app/controllers/api/guidance_controller.rb

```diff
@@ -16,18 +17,32 @@ def release_notes
     end
 
     def page
-      path = params[:page]
-
-      template = {
-        "swagger-api-documentation" => "swagger_api_documentation",
-        "guidance-for-lead-providers" => "guidance_for_lead_providers",
-        "sandbox" => "sandbox",
-        "guidance-for-lead-providers/api-data-states" => "guidance_for_lead_providers/api_data_states",
-        "guidance-for-lead-providers/data-syncing" => "guidance_for_lead_providers/data_syncing",
-        "guidance-for-lead-providers/ids-explained" => "guidance_for_lead_providers/ids_explained",
-      }.fetch(path)
-
-      render "api/guidance/" + template
+      template = "api/guidance/#{params[:page].underscore}"
+
+      if template_exists?(template)
+        render template
+      else
+        render 'errors/not_found', status: :not_found
+      end
+    end
+
+  private
+
+    def guidance_pages
```

Comment:
maybe this logic is better suited to be in a component such as guidance sidebar navigation component? We can pass in the current path and it decides if we should show the sidenav, and the guidance pages are defined there?
Happy to chat through

Reply:
moved into a component